starport/templates/typed/list/stargate/component/x/{{moduleName}}/client/cli/query_{{typeName}}_test.go.plush

```diff
@@ -133,6 +145,9 @@ func TestList<%= TypeName.UpperCamel %>(t *testing.T) {
 		require.NoError(t, net.Config.Codec.UnmarshalJSON(out.Bytes(), &resp))
 		require.NoError(t, err)
 		require.Equal(t, len(objs), int(resp.Pagination.Total))
-		require.Equal(t, objs, resp.<%= TypeName.UpperCamel %>)
+		require.Equal(t,
```

Comment:
We can use `ElementsMatch` here

Reply:
Same comment for the other types.

And also in the Total test in `grpc_query_<name>_test.go` scaffolded files